core.js

```diff
@@ -144,7 +144,13 @@ Runner.prototype.run = function(target, context) {
       var pending = []
       _.each(validationHash, function(validations, key) {
         _.each(validations, function(validation) {
-          pending.push(processItemAsync(runner, validation, key, context).catch(addError(errors, key, validation)))
+          pending.push(processItemAsync(runner, validation, key, context)
+            .then(function(result) {
+              if (_.isBoolean(result) && result === false) {
```

Comment:
`_.isBoolean` is redundant here. `result === false` is suffient (since `false` is of course boolean).